onnx/test/basic_test.py

```diff
@@ -2,12 +2,44 @@
 from __future__ import division
 from __future__ import print_function
 from __future__ import unicode_literals
+
 from onnx.onnx_pb2 import AttributeProto, NodeProto, GraphProto, ModelProto, IR_VERSION
+
+import io
+import onnx
+import tempfile
 import unittest
 
 
 class TestProtobufExists(unittest.TestCase):
 
+    def test_load(self):
+        # Create a model proto.
+        model = ModelProto()
+        model.ir_version = IR_VERSION
+        model_string = model.SerializeToString()
+
+        # Test if input is string
+        loaded_model = onnx.load_from_string(model_string)
+        self.assertTrue(model == loaded_model)
+
+        # Test if input has a read function
+        f = io.BytesIO(model_string)
+        loaded_model = onnx.load(f)
+        self.assertTrue(model == loaded_model)
+
+        # Test if input is a file name
+        try:
+            # Bypass the test if there is no permission
+            f = tempfile.NamedTemporaryFile()
+            with open(f.name, "wb") as file:
+                file.write(model_string)
+            loaded_model = onnx.load(f.name)
+            self.assertTrue(model == loaded_model)
+            f.close()
+        except:
```

Comment:
Doesn't look good in a test.

Reply:
> Whether the name can be used to open the file a second time, while the named temporary file is still open, varies across platforms (it can be so used on Unix; it cannot on Windows NT or later)

https://docs.python.org/2/library/tempfile.html#tempfile.NamedTemporaryFile